src/main/java/org/kainos/ea/resources/JobController.java

```diff
@@ -20,7 +24,7 @@
 @Path("/api")
 public class JobController {
     DatabaseConnector databaseConnector = new DatabaseConnector();
-    JobDao jobDao = new JobDao(databaseConnector);
+    JobDao jobDao = new JobDao();
```

Comment:
Keep an eye out for this, other branches have the job DAO taking a database connector. This could be missed in a merge conflict.

Reply:
Can you check latest commit to see I've fixed this the right way?

Reply:
hello i am going to pull this down and make a couple of changes rq